packages/fast-development-site-react/src/components/site/index.tsx

```diff
@@ -607,6 +605,7 @@ class Site extends React.Component<ISiteProps & IManagedClasses<ISiteManagedClas
                 <ComponentWrapper
                     key={index}
                     index={index}
+                    dir={this.state.locale === "en" ? "ltr" : "rtl"}
```

Comment:
When sites start functioning as true localized sites there will be a lot more locales than "en".  I'd consider, making this more robust to support all locales.  There are 16 regions and about half of them would require ltr over rtl.